src/com/palmergames/bukkit/towny/listeners/TownyPlayerListener.java

```diff
@@ -771,23 +776,80 @@ public void onPlayerChangePlotEvent(PlayerChangePlotEvent event) {
 		}		
 	}
 	
-	/*
+	/**
 	 * onOutlawEnterTown
-	 * - Shows message to outlaws entering towns in which they are considered an outlaw.
-	 */
+	 * - Handles outlaws entering a town they are outlawed in.
+	 **/
 	@EventHandler(priority = EventPriority.NORMAL)
 	public void onOutlawEnterTown(PlayerEnterTownEvent event) throws NotRegisteredException {
-		
-		Player player = event.getPlayer();		
-		WorldCoord to = event.getTo();
-		Resident resident = TownyUniverse.getInstance().getDataSource().getResident(player.getName());
 
-		if (to.getTownBlock().getTown().hasOutlaw(resident))
-			TownyMessaging.sendMsg(player, Translation.of("msg_you_are_an_outlaw_in_this_town", to.getTownBlock().getTown()));
+		Resident outlaw = TownyUniverse.getInstance().getDataSource().getResident(event.getPlayer().getName());
+		Town town = event.getEnteredtown();
+		TownBlock location = event.getTo().getTownBlock();
+		
+		String[] split = new String[0];
+		long cooldown = TownySettings.getOutlawKickCooldown();
```

Comment:
All three of these variables won't be needed after the below changes are made.

Reply:
Changes were made, variables were removed